src/gapp_login/sms/config.py

```diff
@@ -6,10 +8,33 @@
 
 config: Config = load_entry_point("config", Config)
 
+
+def get_sms_provider(settings: str):
+    """Configs used to create a SMS provider instance. Sample:
+    {
+        "params": {
+            "secret_id": "xxx",
+            "secret_key": "xxx",
+            "sms_app_id": "234123",
+            "sms_template_id": "123456"
+            "sms_sign": "HiDay"
+        },
+        "type": "gapp_login.sms.provider.Tencent"
```

Comment:
This is a good idea - would be better if we could make SMS providers plugins - the Tencent one can be built-in, but would be nice if we could support arbitrary providers from 3rd-party libraries.

Reply:
Got it, I'll fix it this weekend.